packages/cli/api-importers/v2-importer-commons/src/converters/abstract/AbstractParameterConverter.ts

```diff
@@ -0,0 +1,178 @@
+                            autogeneratedExamples: {}
+                        },
+                        availability
+                    },
+                    inlinedTypes
+                };
+            case "header":
+                return {
+                    type: "header",
+                    parameter: {
+                        name: context.casingsGenerator.generateNameAndWireValue({
+                            name: this.parameter.name,
+                            wireValue: this.parameter.name
+                        }),
+                        docs: this.parameter.description,
+                        valueType: typeReference ?? AbstractParameterConverter.OPTIONAL_STRING,
+                        env: undefined,
+                        v2Examples: {
+                            userSpecifiedExamples: await this.convertParameterExamples({ context, errorCollector }),
+                            autogeneratedExamples: {}
+                        },
+                        availability
+                    },
+                    inlinedTypes
+                };
+            case "path":
+                return {
+                    type: "path",
+                    parameter: {
+                        name: context.casingsGenerator.generateName(this.parameter.name),
+                        docs: this.parameter.description,
+                        valueType: typeReference ?? AbstractParameterConverter.STRING,
+                        location: "ENDPOINT",
+                        variable: undefined,
+                        v2Examples: {
+                            userSpecifiedExamples: await this.convertParameterExamples({ context, errorCollector }),
+                            autogeneratedExamples: {}
+                        }
+                    },
+                    inlinedTypes
+                };
+            default:
+                return undefined;
+        }
+    }
+
+    protected async convertParameterExamples({
+        context,
+        errorCollector
+    }: {
+        context: AbstractConverterContext<object>;
+        errorCollector: ErrorCollector;
+    }): Promise<Record<string, unknown>> {
+        const parameterExample = this.parameter.example;
+        const parameterExamples = this.parameter.examples;
+        const resolvedParameterExamples: Record<string, unknown> = {};
+        for (const [key, example] of Object.entries(parameterExamples ?? {})) {
+            if (context.isReferenceObject(example)) {
+                const resolved = await context.resolveReference(example);
+                if (resolved.resolved) {
+                    resolvedParameterExamples[key] = resolved.value;
+                }
+            } else if (example != null) {
+                resolvedParameterExamples[key] = example;
+            }
+        }
+        const schemaExampleName = context.generateUniqueName("schemaExample", Object.keys(parameterExamples ?? {}));
```

Comment:
what does `"schemaExample"` do here? may be better to use the parameter name possibley `${paramter}_example`